docs/source/benchmarking.rst

```diff
@@ -71,73 +90,84 @@ Aggregate across simulated communities (when dataset index is `not` unique)
     df = pd.concat(dfs)
     df.to_csv("benchmarks.tsv", sep='\t', index=True, header=True)
 
-Downloading Test Datasets
-=========================
-
-Using the built-in ``autometa`` module
---------------------------------------
-
-Autometa is packaged with a built-in module that allows any user to download any of the available test datasets.
-To use these utilities simply run the command ``autometa-download-dataset``.
+Benchmark classification
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-For example, to download all of the simulated communities reference binning/taxonomy assignments as well as the Autometa
-v2.0 binning/taxonomy predictions:
+Previously we benchmarked the clustering step. Now we are going to benchmark the taxon profiling step. The only difference is that the ``--classification`` flag is provide instead of the ``--clustering`` flag.
 
 .. code:: bash
 
-    # Note: community is the test dataset that was used for clustering or classification. e.g.
-    # choices: 78Mbp,156Mbp,312Mbp,625Mbp,1250Mbp,2500Mbp,5000Mbp,10000Mbp
-    community_sizes=(78Mbp 156Mbp 312Mbp 625Mbp 1250Mbp 2500Mbp 5000Mbp 10000Mbp)
-
-    autometa-download-dataset \
-    --community-type simulated \
-    --community-sizes ${community_sizes[@]} \
-    --file-names reference_assignments.tsv.gz binning.tsv.gz taxonomy.tsv.gz \
-    --dir-path simulated
+    autometa-benchmark \
```

Comment:
Unassigned variables should not be in the documentation. `${community_size}` should be assigned in the same code block.

Reply:
My opinion is that this overly complicated as a  example. I would remove the for loop and analyzing multiple datasets at once. That could be covered in an "Advanced" section.